Enterprise-CMCS/eAPD
# Tbolt/4506 Updates to proposed budget and executive summary

Resolves #4506 

### Description
Updates to table designs

[Chromatic approval ](https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4583)

### Significant changes or possible side effects
* Styling changes for many of the table classes. Possible side-effects: Updating a CSS class has a cascading effect and updates an unintended component/table

### Automated test cases written
No new tests needed for these design changes. Tests were updated where needed.

### Steps to manually verify this change
1. Login to view APDs
2. View an APD with MMIS type
3. Verify Changes are according to ticket + Figma
4. Repeat with an HITECH APD type
5. Since we changed the order of the tables - swapping the federal and state share columns - test that state share data and federal share data is displaying in their correct columns

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
